qe_pre_installation_tasks.yml

```diff
@@ -0,0 +1,12 @@
+---
+#
+# Perform auxiliary tasks before installing Tendrl itself:
+# * update all packages
+#
+- hosts: all
+  remote_user: root
+  tasks:
+    - name: upgrade all packages
+      yum:
```

Comment:
No part of the setup playbooks count with Ubuntu now, so we will have to solve it globally. Thus I don't think it make much sense to add some _if_ to this one step now.